lib/apiary/command/styleguide.rb

```diff
@@ -0,0 +1,190 @@
+# encoding: utf-8
+require 'rest-client'
+require 'rack'
+require 'ostruct'
+require 'json'
+
+require 'apiary/agent'
+require 'apiary/helpers'
+require 'apiary/exceptions'
+
+module Apiary::Command
+  class Styleguide
+    include Apiary::Helpers
+
+    attr_reader :options
+
+    def initialize(opts)
+      @options = OpenStruct.new(opts)
+      @options.fetch        ||= false
+      @options.add          ||= '.'
+      @options.functions    ||= '.'
+      @options.rules        ||= '.'
+      @options.api_key      ||= ENV['APIARY_API_KEY']
+      @options.proxy        ||= ENV['http_proxy']
+      @options.api_host     ||= 'api.apiary.io'
+      @options.vk_url       ||= 'https://z9z93fjwok.execute-api.us-east-1.amazonaws.com/production/validate'
```

Comment:
We know it won't break? Can we add our CNAME in front of it?

Reply:
address is pernament but i'd agree CNAME would be even better - @nadade @abtris - could you set it pls ? (http://voight-kampff-aws.apiary-services.com ?)

Reply:
I'd say it's a blocker - get a domain there

Reply:
@tu1ly CNAME of what to what? :)

Reply:
@nadade 

` https://z9z93fjwok.execute-api.us-east-1.amazonaws.com/production` -> `http://voight-kampff-aws.apiary-services.com`

https://console.aws.amazon.com/apigateway/home?region=us-east-1#/apis/z9z93fjwok/stages/production

Reply:
done, you'll need to wait for AWS to finish the setup